luigi/contrib/dataproc.py

```diff
@@ -163,6 +163,11 @@ def complete(self):
 
     def run(self):
         base_uri = "https://www.googleapis.com/compute/v1/projects/{}".format(self.gcloud_project_id)
+        software_config = {}
+
+        if self.image_version:
+            software_config["imageVersion"] = self.image_version
```

Comment:
This oneliner is much more readable imo:

software_config = { "imageVersion": self.image_version} if self.image_version else {}